--- /usr/src/drivers/Makefile.orig	Sat May 19 16:23:01 2012
+++ /usr/src/drivers/Makefile	Sat May 19 16:23:16 2012
@@ -8,7 +8,7 @@
 SUBDIR = acpi
 .endif
 
-SUBDIR+= ahci amddev atl2 at_wini audio bios_wini dec21140A dp8390 dpeth \
+SUBDIR+= 8ball ahci amddev atl2 at_wini audio bios_wini dec21140A dp8390 dpeth \
 	e1000 filter floppy fxp hello lance log orinoco pci printer \
 	random readclock rtl8139 rtl8169 sb16 ti1225 tty \
 	.WAIT ramdisk .WAIT memory
