--- /usr/src/include/minix/types.h.orig	Sun May 20 22:49:36 2012
+++ /usr/src/include/minix/types.h	Sun May 20 22:49:38 2012
@@ -36,7 +36,7 @@
 typedef u32_t block_t;	   /* block number */
 typedef u32_t bit_t;	   /* bit number in a bit map */
 typedef u16_t zone1_t;	   /* zone number for V1 file systems */
-typedef u16_t bitchunk_t; /* collection of bits in a bitmap */
+typedef u32_t bitchunk_t; /* collection of bits in a bitmap */
 
 /* ANSI C makes writing down the promotion of unsigned types very messy.  When
  * sizeof(short) == sizeof(int), there is no promotion, so the type stays
